tests/slurm-tests/asr_nim/check_results.py

```
@@ -0,0 +1,276 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import re
+import sys
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))  # for utils.py
+from utils import assert_all, soft_assert  # noqa: E402
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)